includes/admin/customers/customers.php

```diff
@@ -801,7 +801,7 @@ function edd_customers_view( $customer = null ) {
 							<input type="hidden" name="customer-id" value="<?php echo esc_attr( $customer->id ); ?>" />
 							<?php wp_nonce_field( 'edd-add-customer-email', 'add_email_nonce', false, true ); ?>
 							<label class="screen-reader-text" for="additional-email"><?php esc_html_e( 'Email Address', 'easy-ditigal-downloads' ); ?></label>
-							<input type="email" name="additional-email" id="additional-email" value="" placeholder="<?php esc_html_e( 'Email Address', 'easy-digital-downloads' ); ?>" />&nbsp;
+							<input type="email" name="additional-email" id="additional-email" value="" placeholder="<?php esc_html__( 'Email Address', 'easy-digital-downloads' ); ?>" />&nbsp;
```

Comment:
```suggestion
							<input type="email" name="additional-email" id="additional-email" value="" placeholder="<?php esc_html_e( 'Email Address', 'easy-digital-downloads' ); ?>" />&nbsp;
```

This was correct before. We want the `input` placeholder to still be visible to give a hint of the field type. The `input` `placeholder` is acting as the "label" for non-vision impaired users and the actual `<label>` value will be read via screen reader for vision impaired users.